src/pages/index.js

```diff
@@ -124,21 +132,19 @@ const section = new Section(
   ".cards__list"
 );
 
-api.getAllData().then((res) => {
-  const user = res[0];
-
-  // Set user info
-  userInfo.setUserInfo(user);
+api
+  .getAllData()
+  .then(([userData, cards]) => {
+    userInfo.setUserInfo(userData);
+    cards.forEach((card) => {
+      // Create new card element to render
+      const cardElement = createCard(card);
 
-  const cards = res[1].reverse();
-  cards.forEach((card) => {
-    // Create new card element to render
-    const cardElement = createCard(card);
-
-    // Add card element to card section to be displayed
-    section.addItem(cardElement);
-  });
-});
+      // Add card element to card section to be displayed
+      section.addItem(cardElement);
+    });
+    section.renderItems();
```

Comment:
* You haven't corrected my remark https://github.com/Conmanz/se_project_aroundtheus/pull/1#discussion_r1373798193 

It should be like this


```js

section.renderItems(cards);
```